scripts/get_coco.sh

```diff
@@ -3,14 +3,14 @@
 # Download command: bash ./scripts/get_coco.sh
 
 # Download/unzip labels
-d='./' # unzip directory
+d='./yolov7' # unzip directory
```

Comment:
```suggestion
d='./' # unzip directory
```

The datasets are meant to be downloaded to the root of the repo. See download command above (`./scripts/get_coco.sh`) and `.gitignore` file while ignores anything under `coco/` in the root.